### PR TITLE
docker: Remove custom entrypoint

### PR DESCRIPTION
**Pull request by IagoAbal**
test plan:
% docker run -v \$(pwd):/src returntocorp/semgrep scan -c p/default .
 #^ should no longer work
% docker run -v \$(pwd):/src returntocorp/semgrep semgrep scan -c p/default .
 #^ should work

PR checklist:

- [x] Purpose of the code is [evident to future readers](https://semgrep.dev/docs/contributing/contributing-code/#explaining-code)
- [x] Tests included or PR comment includes a reproducible test plan
- [x] Documentation is up-to-date
- [x] A changelog entry was [added to changelog.d](https://semgrep.dev/docs/contributing/contributing-code/#adding-a-changelog-entry) for any user-facing change
- [x] Change has no security implications (otherwise, ping security team)

If you're unsure on any of this, please see:

- [Contribution guidelines](https://semgrep.dev/docs/contributing/contributing-code)!
- [One of the more specific guides located here](https://semgrep.dev/docs/contributing/contributing/)

### CODE REVIEW

**Comment by IagoAbal** [comment]
Not a Docker expert, should we just remove `entrypoint.sh` altogether?

File: changelog.d/pa-2642.changed [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Remove custom entry point for returntocorp/semgrep Docker image's, now you must
```
Comment (underyx) [inline changelog.d/pa-2642.changed]:
```suggestion
Remove custom entrypoint for returntocorp/semgrep Docker images, now you must
```

**Review by underyx — APPROVED** [review]

File: entrypoint.sh
Comment (underyx) [inline entrypoint.sh:1]: I'm curious, can't we just delete this whole file now?

Comment (underyx) [reply entrypoint.sh:1]: and remove the `ENTRYPOINT ["/entrypoint.sh"]` line in the Dockerfile

**Comment by underyx** [comment]
> Not a Docker expert, should we just remove `entrypoint.sh` altogether?

yep, I think so

**Pull request merged**